vtds_cluster_kvm/private/scripts/disk_builder.py

```diff
@@ -0,0 +1,545 @@
+#! python
```

Comment:
REVIEWERS: this is where the Disk Builder base class and the two newly added Disk Builder class implementations are defined. It is one of the python libraries sent to the blade during the `prepare` phase.